E1a-ARSE/ALIN01/Dataset/README.md

```diff
@@ -13,27 +13,74 @@ The Following occurrence variations need to be checked and certified in relation
 This test case utilises the following dataset:
 
 - One alignment layout (no geometric representation is required)
-
 - The alignment is composed of one horizontal alignment layout and one vertical alignment layout
-
 - The horizontal layout has the following sequences of elements:
 
   - Line => Clothoid => Circular arc (cw) => Clothoid => Line => Clothoid => Circular arc (ccw) => Clothoid => Line
 
   - Line => Circular arc (cw) => Circular arc (ccw) => Line
 
   - Line => Circular arc (ccw) => Circular arc (cw) => Line
-
 - The vertical layout has the following sequences of elements:
 
   - Line => Circular arc => Line
-
   - Line => Circular arc (cw) => Circular arc (ccw) => Line
-
   - Line => Circular arc (ccw) => Circular arc (cw) => Line
 
+The following tables lists the entities that shall be present in the dataset in addition to the imported entities for model setup :
+
+Alignment:
+
+| Name | Type                   | ObjectPlacement   | Representation |
+| ---- | ---------------------- | ----------------- | -------------- |
+| A1   | IfcAlignment           | IfcLocalPlacement | None           |
+| AH1  | IfcAlignmentHorizontal | IfcLocalPlacement | None           |
+| AV1  | IfcAlignmentVertical   | IfcLocalPlacement | None           |
```

Comment:
I'd not expect ObjectPlacement if there is no Representation.

Reply:
Should be fixed now.